pkg/crcontrollers/reservation/reservationcontroller.go

```diff
@@ -43,6 +45,17 @@ func NewController(client *k8s.KubeClient, log *logrus.Logger, sequentialLVGRese
 func (c *Controller) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&acrcrd.AvailableCapacityReservation{}).
+		WithOptions(controller.Options{
+			// Rater controls timeout between reconcile attempts (if result.Requeue is true)
+			// Default RateLimiter is exponential, which leads to increasing timeout after 100 retries to 5+ mins
+			// Many attempts are expected during processing ACRs with LVG Volumes, so we need to use linear RateLimiter
+			// Here we make first 30 retries with 1.5 sec timeout and then it will be 12 sec forever
+			RateLimiter: workqueue.NewItemFastSlowRateLimiter(
```

Comment:
Is this applicable for all `ctrl.Result{Requeue: true}` attempts ? Maybe it's better to use `ctrl.Result{RequeueAfter: time.Duration}` for processing ACRs with LVG Volumes ?

Reply:
Yes, timeouts would be the same for all errors. We have not so many endpoints in this controller and these changes won't affect them in the short attempts queue.
If we use `ctrl.Result{RequeueAfter: time.Duration}`, timeout will be const on all intervals. It is not good due to many calls to k8s API